css/how-we-write-css.md

```diff
@@ -25,19 +25,22 @@ Contains all **global** SASS variables for your project e.g. colors, fonts, medi
 Contains all **global** SASS functions e.g. em calculation, unit stripping.
 
 #### Mixins
-Contains all **global** SASS mixins, this includes mixin definitions used in the **pattern** and **utility** levels.
+Contains all **global** SASS mixins, this includes mixin definitions used in the **component** and **utility** levels.
 
 #### Base
 A Base rule is applied to an element using an element/type selector, a descendant selector, or a child selector, along with any pseudo-classes. Base styles are related to the basic styles of your product, like typography, reset and global elements like links. Use this level to include any (third-party) resets or normalization css.
 
 #### Components
-Components use classes to map to specific UI elements. They should exist as stand-alone UI components and **never** depend on other components.
+Components use classes to map to specific UI elements.
 
-#### Patterns
-Patterns use classes to apply a reusable design pattern. This level should include any pattern that does not bind itself to a particular UI element. Styles written here can include both structural and cosmetic rules, and apply branding.
+They should be **reusable** and independent of context.
```

Comment:
I'm not sure I 100% agree with this. e.g. a header component, or a main nav. Technically reusable, yes. But practically?